public/js/login.js

```diff
@@ -35,6 +35,7 @@ $(document).ready(function() {
           window.location.replace("/sightings");  ///Change this
           // If there's an error, log the error
         })
+
```

Comment:
? Why empty string? I'm ok with this , but it seems for me that catching error right after waas more easy to understand

Reply:
i am deleting this request